### ISSUE TITLE
[Server] Foot mounting calib should work

### ISSUE THREAD

**Issue by TheButlah** [SlimeVR/SlimeVR-Server#582]
@ButterscotchV said that the mounting calibration doesn't work for feet right now. This is because there is no good foot position to stand in that can be one at the same time as the skiing pose (too hard to stand on tip toes or heels).

**Comment by evg-zhabotinsky** [comment]
Wouldn't it work if you actually get on your knees? Haven't checked, when I had the IMUs backwards the thought didn't occur, and after fixing that I never needed to calibrate mounting for feet, they are very forgiving in that regard.

**Comment by ButterscotchV** [comment]
> Wouldn't it work if you actually get on your knees? Haven't checked, when I had the IMUs backwards the thought didn't occur, and after fixing that I never needed to calibrate mounting for feet, they are very forgiving in that regard.

Yes that could work? Seems a tiny bit iffy though. Currently my best thought is either sitting with your legs out in front, laying down, or legs kicked out in a chair as the same option (anything to tilt the foot backwards)

**Comment by Stermere** [comment]
Another option would be to do it one foot at a time so that it can still be done standing up, this comes with its own drawbacks as well.
